torch/_dynamo/eval_frame.py

```diff
@@ -364,6 +364,7 @@ def get_compiler_config():
                 not in ["_call_impl", "_wrapped_call_impl", "_lazy_forward"]
             )
             and filename not in DONT_WRAP_FILES
+            and isinstance(fn, types.BuiltinFunctionType)
```

Comment:
This is not correct. We are handling these functions/objects which locate at torch builtin modules at here, since they were possible in skipfiles, so we have to add an extra frame to trigger torch.compile.

Reply:
It looks like there is another place where Pytorch also checks if a function was created in exec/eval: https://github.com/pytorch/pytorch/blob/e0d2c24de1e7726fbb161e7b5c91063bf97d6565/torch/utils/_traceback.py#L60

I changed the check here to use that same way of testing for exec/eval instead.